src/docs/santiment-queries/metric-tables/index.md

```diff
@@ -0,0 +1,374 @@
+
+## Tables
+
+---
+
+### daily_metrics_v2
+
+Table `daily_metrics_v2` is the most basic table. It stores a signle metric value for each asset for each day. I.e. in that tables stored metrics that have exactly 1 value per day.
+
+Here's an example how to fetch `daily_active_addresses` for `bitcoin` using `daily_metrics_v2`
+
+```SQL
+SELECT
+    dt,
+    get_asset_name(asset_id) AS asset,
+    get_metric_name(metric_id) AS metric,
+    value
+FROM daily_metrics_v2 FINAL
+WHERE
+    asset_id = get_asset_id('bitcoin') AND
+    metric_id = get_metric_id('daily_active_addresses') AND
+    dt >= toDateTime('2020-01-01 00:00:00')
+LIMIT 2
+```
+Test in [Queries](https://queries.santiment.net/query/dict-example-with-func-413)
+
+---
+
+### intraday_metrics
+
+Table `intraday_metrics` is very similar to `daily_metrics_v2` but stores metrics with more than one value per day. In most cases, these metrics have a new value every 5 minutes.
+
+Here's an example how to get 5-minute prices for ethereum:
+
+```SQL
+SELECT
+    dt,
+    get_asset_name(asset_id) AS asset,
+    get_metric_name(metric_id) AS metric,
+    value
+FROM intraday_metrics FINAL
+WHERE
+    asset_id = get_asset_id('ethereum') AND
+    metric_id = get_metric_id('price_usd') AND
+    dt >= toDateTime('2024-01-01 00:00:00')
+ORDER BY dt DESC
+LIMIT 5
+```
+
+Test in [Queries](https://queries.santiment.net/query/price-example-478)
+
+---
+
+### labeled_intraday_metrics_v2
+
+Table `labeled_intraday_metrics_v2` stores metrics for a given `blockchain` and `label_id`. The data for that ID is stored in the `label_metadata` table. Blockchain column contains the string for blockchain name, e.g. `bictoin`, `ethereum`, etc.
+
+For instance, you can retrieve ethereum balance of all centralized exchange addresses with the query:
+
+```SQL
+SELECT
+    dt,
+    value AS balance
+FROM labeled_intraday_metrics FINAL
+WHERE
+    dt >= toDateTime('2024-01-01 00:00:00')
+    AND label_id = dictGet('labels_by_fqn', 'label_id', 'santiment/centralized_exchange:v1')
+    AND metric_id = dictGet('metrics_by_name', 'metric_id', 'combined_labeled_balance')
+    AND asset_id = dictGet('assets_by_name', 'asset_id', 'ethereum')
+    AND blockchain = 'ethereum'
+ORDER BY dt DESC
+LIMIT 5
+```
+
+Test in [Queries](https://queries.santiment.net/query/cex-balance-example-479)
+
+---
+
+### daily_label_based_metrics
+
+Table `intraday_metrics` also has `label` and `owner` columns so that metrics stored in that table correspond to a particular label and owner.
+
+> Note that this table uses old way to filter labels (plain label/owner instead of label_fqn).
+
+This table could be used to retrieve [labelled exchange](/metrics/labeled-exchange) metrics under names: 
+
+* `labelled_active_deposits`
+* `labelled_deposit_transactions`
+* `labelled_active_withdrawals`
+* `labelled_withdrawal_transactions` 
+
+or NFT [trading volume](metrics/nft-trade-volume-usd) and [trades count](metrics/nft-trades-count) metrics.
```

Comment:
Those links doesn't work. Maybe the metric has been renamed or removed?

Reply:
Should work once in master, Kate added these articles in a separate branch https://github.com/santiment/academy/pull/393